src/Internal.AspNetCore.Sdk/Internal.AspNetCore.Sdk.nuspec

```diff
@@ -9,6 +9,7 @@
     <copyright>$copyright$</copyright>
     <dependencies>
       <dependency id="Microsoft.AspNetCore.BuildTools.ApiCheck" version="$apicheckversion$" />
+      <dependency id="SourceLink.Create.CommandLine" version="2.2.1" />
```

Comment:
I'm hesitant to add new third-party dependencies to our build unless we need to. The requirements of build-from-source will mean we need to either (1) clone, compile and package SourceLink.Create.CommandLine  or (2) conditionally exclude this tool when building from source.

As an alternative, could source link be done as a post-processing step with existing symbols packages? It would avoid these issues.

cc @mikeharder

Reply:
I used this package because of https://github.com/aspnet/Universe/issues/553#issuecomment-328742192. The code required here isn't super complicated, we could do it ourselves without having to edit PDB's afterwards.

Reply:
I think we should look into that. Putting this in Internal.AspNetCore.Sdk.nuspec makes it hard to conditionally exclude or skip it.